src/usb_iss/usb_iss.py

```diff
@@ -49,7 +49,14 @@ def __init__(self, dummy=False):
         self.io = IO(self._drv)
         self.spi = SPI(self._drv)
         self.serial = Serial(self._drv)
-
+    
+    def get_port(self, vid=defs.VID_USB_ISS, pid=defs.PID_USB_ISS):
+        plist = list(serial.tools.list_ports.comports())
+        for p in plist:

```

Comment:
list_ports returns an iterable, so you should be able to say `for port in list_ports.comports():`

Reply:
Nit: descriptive variable names (port) are better than single letters (p).